src/Uno.UI/UI/Xaml/Controls/ContentPresenter/ContentPresenter.cs

```diff
@@ -955,7 +955,8 @@ private void SynchronizeVerticalContentAlignment(VerticalAlignment? alignment =
 
 		protected override void OnBackgroundChanged(DependencyPropertyChangedEventArgs e)
 		{
-			base.OnBackgroundChanged(e);
+			// Don't call base, the UpdateBorder() method handles drawing the background.
+			// base.OnBackgroundChanged(e);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5135557)

Reply:
Left commented-out on purpose